Jetcaster/app/src/main/java/com/example/jetcaster/ui/MainActivity.kt

```diff
@@ -30,10 +40,30 @@ class MainActivity : ComponentActivity() {
         // This app draws behind the system bars, so we want to handle fitting system windows
         WindowCompat.setDecorFitsSystemWindows(window, false)
 
+        /**
+         * Flow of [FoldableInfo] that emits every time there's a change in the windowLayoutInfo
+         */
+        val foldableInfo = windowInfoRepository().windowLayoutInfo
```

Comment:
I wish this could be a data source, but obviously needs a window... would this be repeated in every activity? do we want devs to copypaste this?